appinfo/info.xml

```diff
@@ -25,7 +25,7 @@ The First run wizard can be customized to meet specific design goals, or to chan
 			<id>firstrunwizard-about</id>
 			<name>About</name>
 			<route></route>
-			<order>5</order>
+			<order>65</order>
```

Comment:
Any reason to not use just 6? 

https://github.com/nextcloud/firstrunwizard/pull/696/files

Reply:
A typo :see_no_evil: